<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin2</class>
 <widget class="QWidget" name="admin2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="23" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Updated Item Quantity</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="31" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Updated Item price ($)</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Item Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>New Item Name</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Price ($)</string>
      </property>
     </widget>
    </item>
    <item row="41" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Commit Changes</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Refresh Inventory</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Add Item</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
    <item row="26" column="0">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="34" column="0">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="3" column="1" rowspan="40">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
